src/style.css

```
@@ -1,3 +1,83 @@
+html {
+  font-family: Roboto, sans-serif;
+  font-weight: 500;
+  font-style: normal;
+}
+
 body {
   margin: 0;
 }
+
+:root {
+  --main-blue: #00acdc;
```

```suggestion
  --main-color: #00acdc;
```
If in the future the main color will be red you have to change name in all places